packages/e/eopkg/package.yml

```diff
@@ -0,0 +1,88 @@
+name       : eopkg
+version    : 4.0.0
+release    : 1
+source     :
+    - git|https://github.com/getsolus/eopkg : c1b7380b5796fdae60adc7bfa1012a8a4104e7df
+    - git|https://github.com/getsolus/PackageKit.git : bc00dd23346676dc333bdb70c2fff4f0a680b688
+homepage   : https://github.com/getsolus/eopkg
+license    : GPL-2.0-or-later
+component  : system.utils
+summary    : UNSUPPORTED & EXPERIMENTAL (reserved for Solus Staff testing)
+description: |
+
+    WARNING: This package is **unsupported**, **experimental and may **break your system!!!**
+
+    It is included at this early stage to make it easier for Solus Staff to test and debug it.
+
+    If you switch back and forth between using eopkg4-bin (py3) and legacy eopkg (py2),
+    do the following for the version you are switching _to_:
+
+    - run `sudo eopkg update-repo --force` or `sudo eopkg4-bin update-repo --force` (regenerates the pickle cache)
+    - run `sudo eopkg rdb` or `sudo eopkg4-bin rdb` (keeps package databases in sync)
+
+    Experienced contributors with packaging and bugreporting experience are welcome to help us test and report (non-duplicate!) problems here:
+
+    https://github.com/getsolus/packages/issues/1316
+
+strip      : no
+debug      : no
+builddeps  :
+    - pkgconfig(packagekit-glib2)
+    - pkgconfig(python3)
+    - iksemel-devel
+    - nuitka
+    - patchelf
+    - python-build
+    - python-installer
+    - python-magic
+    - python-packaging
+    - python-wheel
+    - python-zstandard
+    - tree
+rundeps    :
+    - iksemel
+setup      : |
+    # NOTE: please leave the grep commands in for visibility purposes during build. Thanks. /ermo
+    grep -Hn version pisi/__init__.py
+    # append the git ref to __version__ on a new line
+    gawk -i inplace 'BEGIN { "git rev-parse --short HEAD" | getline gitref } { print }; /__version__ = / { printf "%s %s\n", $1, "+= \" (" gitref ")\"" }' pisi/__init__.py
+    grep -Hn version pisi/__init__.py
+
+    %python3_setup
+build      : |
+    # This prevents the dynamic loader and glibc from attempting to resolve shared objects via hwcaps,
+    # and effectively forces nuitka to build with x86_64-v1 (OG AMD64). Note that more flags may need
+    # to be added in the future to guard against either x86_64-v2, x86_64-v3 or x86_64-v4 hwcaps being used.
+    export GLIBC_TUNABLES=glibc.cpu.hwcaps=-AVX
+
+    # We're not actually using self-execution. In this case, eopkg is using the -c flag as shorthand for --component, rather than for passing the program as a string (as is default python behavior).
+    nuitka3 --onefile --include-module=dbm.gnu --show-scons --no-deployment-flag=self-execution $workdir/eopkg-cli
+    nuitka3 --onefile --include-module=dbm.gnu $sources/PackageKit.git/backends/eopkg/eopkgBackend.py
+install    : |
+    # install the normal stuff
+    %python3_install
+
+    # useful to give insight for the for loop below
+    #tree $installdir/usr/bin
```

Comment:
nit: duplicated further down

Reply:
The first one is to be able to distinguish between what the %python3_install macro installs.

The final one is about being able to confirm what was installed in totality after the install section is complete (and whether it matches expectations)